.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
No need to set this IMO since node 14 is still very common

Reply:
This is not going to affect the users of the library, only for the people contributing to this repo. But I'll remove anyways, you can add it back if you want this in future 👍

Reply:
Ah yea. I was talking about contributors as well. Even within contributors, people use node 14 often.